wgengine/router/router_linux.go

```diff
@@ -1400,6 +1400,15 @@ func normalizeCIDR(cidr netip.Prefix) string {
 // The function calls cleanUp for both iptables and nftables since which ever
 // netfilter runner is used, the cleanUp function for the other one doesn't do anything.
 func cleanUp(logf logger.Logf, interfaceName string) {
+	if distro.Get() == distro.Synology {
```

Comment:
No clue. Anybody test?

Reply:
We don't add any rules if `distro.Get() == distro.Synology` though so there should never be a need to clean up our routes then

Reply:
just theoretically I wouldn't expect QNAP to be a problem because we don't turn netfilter off for QNAP so if it was able to run in tun mode before then it would have been creating iptables rules already